builder/rootfs/bin/boot

```diff
@@ -14,14 +14,16 @@ fi
 [[ $DEBUG ]] && set -x
 
 # configure etcd
-export ETCD_PORT=${ETCD_PORT:-4001}
-export ETCD="$HOST:$ETCD_PORT"
+# if ETCDCTL_PEERS isn't already set, set it
+if [ -z "$ETCDCTL_PEERS" ]; then
+	export ETCDCTL_PEERS="http://${ETCD_HOST:-${HOST:-127.0.0.1}}:${ETCD_PORT:-4001}"
```

Comment:
I know at some point we may want to be able to support etcd clusters that are also backed by SSL. Can we make the protocol (or the URL as a whole) configurable?